### ISSUE TITLE
Gnumeric.workbooks() doesn't exist in Gnumeric 1.12.53

### ISSUE THREAD

**Issue by justaquestion** [amir-hanna/gnumeric#1]
Thank you for your great python helper methods.
Now I use Gnumeric 1.12.53 and Gnumeric.workbooks() simply does not seem to exist anymore.
Do you have any idea how to create the 'obj' with this Gnumeric version to call for example:
`Gnumeric['cell']('value',obj)`
?
```
>>> dir(Gnumeric)
['FALSE', 'GnumericError', 'GnumericErrorDIV0', 'GnumericErrorNA', 'GnumericErrorNAME', 'GnumericErrorNULL', 'GnumericErrorNUM', 'GnumericErrorREF', 'GnumericErrorVALUE', 'TRUE', '__doc__', '__loader__', '__name__', '__package__', '__spec__', 'functions']
```

**Comment by amir-hanna** [comment]
Hello,

Thank you for pointing this out. Are you trying this on a machine where python 3 is the default installation?

I can see there was an open bug about this 2 years ago:

https://gitlab.gnome.org/GNOME/gnumeric/-/issues/419

Anyway, if I find a solution, I shall get back to you as soon as I can.

Thank you
Amir Hanna
________________________________
From: justaquestion ***@***.***>
Sent: Sunday, December 18, 2022 1:12:31 PM
To: amir-hanna/gnumeric ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [amir-hanna/gnumeric] Gnumeric.workbooks() doesn't exist in Gnumeric 1.12.53 (Issue #1)


Thank you for your great python helper methods.
Now I use Gnumeric 1.12.53 and Gnumeric.workbooks() simply does not seem to exist anymore.
Do you have any idea how to create the 'obj' with this Gnumeric version to call for example:
Gnumeric['cell']('value',obj)
?

>>> dir(Gnumeric)
['FALSE', 'GnumericError', 'GnumericErrorDIV0', 'GnumericErrorNA', 'GnumericErrorNAME', 'GnumericErrorNULL', 'GnumericErrorNUM', 'GnumericErrorREF', 'GnumericErrorVALUE', 'TRUE', '__doc__', '__loader__', '__name__', '__package__', '__spec__', 'functions']

**Comment by justaquestion** [comment]
Yes, I have Python 3 on Arch Linux. Nothing urgent, I have completed some Python courses, and I am just trying out things. For the moment pyspread looks the most promising and of course Jupyter.
